builtin/settingtypes.txt

```diff
@@ -268,6 +268,15 @@ performance_tradeoffs (Tradeoffs for performance) bool false
 #    Adds particles when digging a node.
 enable_particles (Digging particles) bool true
 
+#   Tint sunlight orange during dusk and dawn.
+tinted_sunlight (Tinted sunlight) bool false
+
+#   The color of light that sources like lamps, torches etc. give off.
+#   -   default: Neutral, white light.
+#   -   medium: Slightly warmer color than default.
+#   -   warm: Warm (yellowish-orangy) light.
```

Comment:
both of these seem to conflict with #14091 (but haven't checked closely)
unless they only apply to the dynamic shadows/light. still then the question is "shouldn't this be up to the API user, instead of the client?"

Reply:
The tinted sunlight thing does, artificial light doesn't. I suppose that this could indeed be API controlled...